pypicloud/views/api.py

```diff
@@ -106,9 +106,21 @@ def download_package(context, request):
         package, data = fetch_dist(request, dist.name, source_url)
         disp = CONTENT_DISPOSITION.tuples(filename=package.filename)
         request.response.headers.update(disp)
+        if request.registry.max_age:
```

Comment:
I think it's safe to include the header even if `max_age = 0` (as you're doing below)